app/Http/Controllers/Conversation/ConversationReplyController.php

```
@@ -0,0 +1,96 @@
+                'message' => 'Failed to create reply',
+                'errors'  => ['body' => ['Unable to save reply. Please try again.']],
+            ], 422);
+        }
+    }
+
+    private function generateDefaultAvatar($user): string
+    {
+        $name = $user->name ?? $user->username;
+
+        return 'https://ui-avatars.com/api/?name='.urlencode($name).'&background=667eea&color=fff&size=128';
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'generateDefaultAvatar'. (undefined)

(UnusedPrivateMethod)